### PR TITLE
Daemon integration

### LINKED ISSUE #123: [Networking] - System State Daemon and Hooks
### Description

Right now, flashing can occur ay any time irregardless of current system state.

### Acceptance Criteria

Driving state is consistently detected and consistenty verifiable, but less importantly does recover the functionality of the safe state 100% of the time.

### Proposed Solution

Read from MQTT such that when the car is in an active, a driving state is triggered.  This should be sent to MQTT as well.  When driving state is triggered various actions can occur.  When driving state is over these actions will be cleanly undone.

The only action this PR should implement is a way to power down pi debug probes when driving mode is entered.

So each plugin should be defined with three functions: on_drive, on_safe, and update_drive. Update_drive should query statuses of the like and ensure rules have been obeyed, not required by all plugins.

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

Integrates the Odysseus-Daemon repository.

## Notes

Currently uses onboard storage, should be improved.
ffmpeg untested.

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] Requires fix of https://bugs.buildroot.org/show_bug.cgi?id=16279
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #123

### CODE REVIEW

File: odysseus_tree/package/calypso/calypso.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CALYPSO_VERSION = 74d3aa1b0595d8f860e0e7d2f7b58d3711c50c27
+CALYPSO_VERSION = e325f41498f678dec023f3f3f603cb386830f643
```
Comment (harrison-e) [inline odysseus_tree/package/calypso/calypso.mk:1]: This is the hash you were talking about?

**Review by harrison-e — APPROVED** [review]
Cool beans!!

File: odysseus_tree/package/calypso/calypso.mk (hunk 1 above)
Comment (jr1221) [reply odysseus_tree/package/calypso/calypso.mk:1]: yup.  cursed thing lol.
